content/plotly/concepts/express/terms/pie/pie.md

```diff
@@ -0,0 +1,48 @@
+# Plotly Pie Charts
+
+## .pie()
+
+In python, we can use *.pie()* from the *plotly library* to display data in the form of a pie chart.
+This is such a simple tool to use and can transform data into visually apealing ways for readers
+to get a better idea about the relationships between variables.
+
```

Comment:
After the introduction paragraph, use the `## Syntax` block to provide the syntax of the `.pie()` method. Additionally, describe the parameters in the syntax in the following format:
- `parameter1`: Its description
- `parameter2`: its description
- `parametern`: its description